packages/lists/src/components/StaticList/examples/StaticTable.Example.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { StaticList } from '../StaticList';
```

Comment:
Note to self to resolve absolute imports and `lib` / `dist` linking with `tsconfig`

Reply:
Resolved in https://github.com/OfficeDev/office-ui-fabric-react/pull/9537/commits/cf01e1d44ff56d130325cfcb2fcbc5255b2b57c1